tests/karma/BookNavigator/volumes/volumes-provider.test.js

```diff
@@ -56,6 +59,22 @@ describe('Volumes Provider', () => {
     expect(provider.component).to.exist;
   });
 
+  describe('sorting cycles - render headerIcon actionButton', async () => {
```

Comment:
Hmmm, I'm not seeing this test in the karma output 😬 Could it be this isn't even getting run? Actually, I'm not seing `VolumesProvider` in the karam output at all.

Uh oh. In jest, `describe` can't be `async`. Could it be the same in karma? I'm noticing https://github.com/internetarchive/bookreader/blob/master/tests/karma/BookNavigator/book-navigator.test.js uses `it` when it has `async`. It might be the case none of these tests are running 😬

Reply:
got it working now, changed `describe` to `it`